### DISCUSSION TITLE
HELP NEEDED

### DISCUSSION THREAD

**Discussion by vkassingh** [openebs/openebs#3261] [Q&A] [answered]
Hi Everyone, I am new to open source world! I want to work towards improving the user end documentation. 

So please suggest some projects/organizations which needs a lot of improvement in their documentation? or how can start my journey for the same?

thank you

**Comment by kmova** [comment, marked as answer]
Hi @vkassingh - you can start by helping with looking at this documentation. 
- https://github.com/openebs/cstor-operators
- https://github.com/openebs/zfs-localpv

There is a larger docs that we need to work on as well under: https://github.com/openebs/openebs-docs
